feast.patch

```diff
@@ -0,0 +1,552 @@
+diff --git a/sdk/python/feast/feature_store.py b/sdk/python/feast/feature_store.py
+index 9655c5d1..010bfded 100644
+--- a/sdk/python/feast/feature_store.py
++++ b/sdk/python/feast/feature_store.py
+@@ -143,7 +143,7 @@ class FeatureStore:
+         """
+         registry_config = self.config.get_registry_config()
+         registry = Registry(registry_config, repo_path=self.repo_path)
+-        registry.refresh()
++        registry.refresh(project=self.project)
+ 
+         self._registry = registry
+ 
+@@ -511,6 +511,38 @@ class FeatureStore:
+                 table, project=self.project, commit=False
+             )
+ 
++        add_odfvs = [o.name for o in odfvs_to_update]
++        add_feature_tables = [ft.name for ft in tables_to_update]
++
++        add_entities = [{
++            'name': ent.name,
++            'data_type': ent.value_type.name
++            }
++            for ent in entities_to_update]
++
++        add_feature_views = [{
++            'name': fv.name,
++            'entities': fv.entities
++        }
++        for fv in views_to_update]
++
++        add_feature_services = [{
++            'name': fs.name,
++            'features': fs.features
++        }
++        for fs in services_to_update]
++
++        infra_update = {
```

Comment:
In feast main branch, they have a way to calculate diff. Is it something new? or, just refactored?

Reply:
https://github.com/feast-dev/feast/tree/master/sdk/python/feast/diff